default/scripting/buildings/buildings.macros

```diff
@@ -104,3 +104,22 @@ SPECIES_LIKES_OR_DISLIKES_BUILDING_STABILITY_EFFECTS
                     )^(-0.5)
                 ) * [[POLICY_LIKE_DISLIKE_SCALING]] * SpeciesContentOpinion species = Target.Species name = ThisBuilding
 '''
+
+DO_STARLANE_BORE
+'''AddStarlanes endpoint = MinimumNumberOf
+       number = 1
+       sortkey = DirectDistanceBetween object = Source.ID object = LocalCandidate.ID
+       condition = And [
+           System
+           CanAddStarlanesTo condition = Source
```

Comment:
Nexus had a maximum length, Bore did not. I think they should both do the same, so if there shall be a limits, we add it here (and in BORE_POSSIBLE below)